### ISSUE TITLE
Nesting img tags inside a p tag produces unexpected result

### ISSUE THREAD

**Issue by adam-buckley** [archriss/react-native-render-html#189]
### Is this a bug report or a feature request?

Bug report

### Have you read [the guidelines](https://github.com/archriss/react-native-render-html/blob/master/CONTRIBUTING.md) regarding bug report?

Yes

### Have you read [the documentation](https://github.com/archriss/react-native-render-html/blob/master/README.md) in its entirety?

I've been at this for hours, so yes

### Have you made sure that your issue hasn't already been reported/solved?

Yes

### Is the bug specific to iOS or Android? Or can it be reproduced on both platforms?

Currently only testing on Android

### Is the bug reproductible in a production environment (not a debug one)?

Yes (without debugger warnings it's hard to be 110% sure of this one)

### Have you been able to reproduce the bug in [the provided example](https://github.com/archriss/react-native-render-html/tree/master/Demo)?

Yes

### Environment

  Environment:
    React: 16.4.1
    React native: 0.55.4
    react-native-render-html: 3.10.0

  Target Platform:
    Android (9.0)

### Steps to Reproduce

1. Add 3 nested images inside a paragraph tag and give the p tag a class:
```html
<p class="centered">
  <img src="any.png" />
  <img src="any.png" />
  <img src="any.png" />
</p>
```
2. Add a classesStyles property to the react-native-renderer-html which centres the paragraph tag:
```jsx
<HTML classesStyles={{"centered": {textAlign: "center"}}} ... ></HTML>
```
3. Run and app and notice an error: "Invalid props.style key `textAlign` supplied to `View`"

### Expected Behavior

I got this issue because our app is receiving HTML from a backend where content is set with a WSYWIG editor. In the editor, I placed three images next to each other and they display fine in our web backend. But once supplied to the app and html renderer they now display stacked. The WSYWIG editor (Quill) attaches a custom class for centering text, so I added a classesStyle for this custom class only to have the debugger warning show up. It took me a while to work out that it was the images causing the issue.

TL;DR: I expected to see the images side by side with no debugger warnings.

### Actual Behavior

As mentioned above, the images stacked on top of each other and the debugger error warned me that I can't use textAlign: "center" on a View. It looks like the renderer is rendering \<Text> elements when it is only text in a paragraph, but changing to a \<View> when images are included.

Screenshot: 
![screenshot_20181004-153131](https://user-images.githubusercontent.com/1585392/46455343-f5cf9a00-c7ed-11e8-9282-50b124876064.png)

### Reproducible Demo

https://snack.expo.io/ry-XqX75m

Thank you for considering the issue, I've spent a fair bit of time trying to make custom renderers to work around this to no avail, any advice is greatly appreciated.
Adam

Edit: sorry I forgot the screenshot

**Comment by Exilz** [comment]
This is quite a tricky case.

On one hand, you can't just render 3 `<Image>` inside a `<Text>` in react-native, especially on android.
In order to fix this, render-html checks the children of your `<p>`, since it only finds some images, it switches your container to a `View`. As you guessed, this is why your warning is displayed.

On the other hand, you can't just use `textAlign: center` to display your images side by side.

What you could try is to create a custom renderer that's used when you encounter your `centered` class. If so, you should be able to render a `<View>` with `{ flex: 1, flex-direction: 'row' }` and completely discard your `textAlign: center`.
